packages/wasm/src/services/credential/index.test.js

```diff
@@ -401,7 +401,7 @@ describe('Credential Service', () => {
           },
         }),
       ).toBe(
-        '0xa632a41f2fbdb681c14b33daae4fcc46af41661b90b35c4ac1545c9bebf0d7cc',
+        'dock:accumulator:0xa632a41f2fbdb681c14b33daae4fcc46af41661b90b35c4ac1545c9bebf0d7cc',
```

Comment:
do we need to support/test both formats (i.e. both with the prefix and without) so we ensure support of old credentials?

Reply:
the prefix didn't change in the credential, credentials always had prefix. The SDK used to require the registryId without prefix, so we had to remove that when generating the presentation. Now that's not needed anymore